### PR TITLE
Update JuMP-dev schedule

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

File: _posts/2022-11-20-jump_dev_2023.md [hunk 1]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+ * Afternoon: other JuliaCon talks
 
 #### July 28
 
-* All day: JuMP-dev talks
+A mix of JuMP-dev and JuliaCon.
+
+ * 09:00: JuliaCon keynote
+ * 10:00: Lessons from OR-Tools, Ross Anderson
+ * 10:30: Optimization in the context of relational database systems, Chris Coey
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:62]:
```suggestion
 * 10:30: Constraint programming using MOI's next-gen nonlinear interface, Chris Coey
```

File: _posts/2022-11-20-jump_dev_2023.md [hunk 2]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+ * Afternoon: other JuliaCon talks
 
 #### July 28
 
-* All day: JuMP-dev talks
+A mix of JuMP-dev and JuliaCon.
+
+ * 09:00: JuliaCon keynote
+ * 10:00: Lessons from OR-Tools, Ross Anderson
+ * 10:30: Constraint programming using MOI's next-gen nonlinear interface, Chris Coey
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:62]: @chriscoey updated

File: _posts/2022-11-20-jump_dev_2023.md [hunk 3]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+ * 10:40: ConstraintLearning: Ever wanted to learn more about constraints? Jean-François Baffier
+ * 10:50: Fast Convex Optimization with GeNIOS.jl, Theo Diamandis
+ * 11:00: Polynomial Optimization, Benoît Legat
+ * 11:30: Computable General Equilibrium (CGE) Models in Julia JuMP, Mitch Phillipson
+ * 11:40: Nested approaches for multi-stage stochastic planning problem,s Alireza Shefaei
```
Comment (jd-foster) [inline _posts/2022-11-20-jump_dev_2023.md:67]:
```suggestion
 * 11:40: Nested approaches for multi-stage stochastic planning problems, Alireza Shefaei
```

File: _posts/2022-11-20-jump_dev_2023.md [hunk 4]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+ * 10:30: Morning break
+ * 10:50: The state of JuMP, Oscar Dowson and Miles Lubin
+ * 11:20: Multi-objective optimization with JuMP, Xavier Gandibleux
+ * 11:30: Debugging JuMP optimization models using graph theory, Robert Parker
+ * 11:40: Plasmo.jl and MadNLP.jl-A Framework for Graph-Based Optimization, David Cole
+ * 11:50: Constructing Optimal Optimization Methods using BnB-PEP, Shuvomoy Das Gupta
+ * 12:00: Solving the merchant collocated facilities with JuMP, Jose Daniel Lara
+ * 12:10: How JuMP enables abstract energy system models, Stefan Strömer
+ * 12:20: TimeStruct.jl: multi horizon time modelling in JuMP, Truls Flatberg
+ * 12:30: Designing a flexible energy system model using multiple dispatch
```
Comment (jd-foster) [inline _posts/2022-11-20-jump_dev_2023.md:51]: Missing a speaker?

Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:51]:
```suggestion
 * 12:30: Designing a flexible energy system model using multiple dispatch, Lars Hellemo and Julian Straus
```

File: _posts/2022-11-20-jump_dev_2023.md [hunk 5]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+A mix of JuMP-dev and JuliaCon.
+
+ * 09:00: JuliaCon keynote
+ * 10:00: Lessons from OR-Tools, Ross Anderson
+ * 10:30: Constraint programming using MOI's next-gen nonlinear interface, Chris Coey
+ * 10:40: ConstraintLearning: Ever wanted to learn more about constraints? Jean-François Baffier
+ * 10:50: Fast Convex Optimization with GeNIOS.jl, Theo Diamandis
+ * 11:00: Polynomial Optimization, Benoît Legat
+ * 11:30: Computable General Equilibrium (CGE) Models in Julia JuMP, Mitch Phillipson
+ * 11:40: Nested approaches for multi-stage stochastic planning problems, Alireza Shefaei
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:67]:
```suggestion
 * 11:40: Nested approaches for multi-stage stochastic planning problems, Alireza Shefaei and Edo Abraham
```

File: _posts/2022-11-20-jump_dev_2023.md [hunk 6]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+ * 11:50: Stochastic programming application for LNGC logistics, Guilherme Bodin
+ * 12:00: When Enzyme meets JuMP: a tour de ronde, François Pacaud
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:69]:
```suggestion
 * 12:00: When Enzyme meets JuMP: a tour de ronde, François Pacaud and Michel Schanen
```

File: _posts/2022-11-20-jump_dev_2023.md [hunk 7]
```diff
@@ -34,12 +36,41 @@ and JuMP-dev specific events are scheduled on July 27-29.
...
+ * 13:30: Lunch
+ * Afternoon: other JuliaCon talks
 
 #### July 28
 
-* All day: JuMP-dev talks
+A mix of JuMP-dev and JuliaCon.
+
+ * 09:00: JuliaCon keynote
+ * 10:00: Lessons from OR-Tools, Ross Anderson
```
Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:61]:
```suggestion
 * 10:00: JuMP-dev keynote: Lessons from OR-Tools, Ross Anderson
```

Comment (mlubin) [inline _posts/2022-11-20-jump_dev_2023.md:61]: Highlight this as an invited talk?

Comment (odow) [inline _posts/2022-11-20-jump_dev_2023.md:61]:
```suggestion
 * 10:00: Invited keynote: Lessons from OR-Tools, Ross Anderson
```

**Pull request merged**